LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2024 Adam Fowler
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
This doesn't quite fix the issue right? The license would still be Apache2 for a new user's project

Reply:
We can now revert this right? Now that the license isn't copied

Reply:
In theory the license shouldn't have my name in it. The license says it is applied by including the copyright in a file header.